utils/dockerutil.py

```diff
@@ -232,6 +232,17 @@ def get_hostname(self, use_default_gw=True, should_resolve=False):
 
         return self._default_gateway
 
+    def get_host_tags(self):
+        tags = []
+        version = self.client.version()
+        if version and 'Version' in version:
+            tags.append('docker_version:%s' % version['Version'])
+        else:
+            log.debug("Could not determine Docker version")
+        if self.is_swarm():
+            tags.append('docker_swarm:active')
```

Comment:
then let's add `inactive` as well? This way we could differentiate between users for whom we didn't collect it (error, old agent version) and those for whom we're sure it's inactive.